src/main/java/com/gachigang/ontherun/persistence/entity/Product.java

```diff
@@ -32,7 +30,7 @@ public class Product {
 
     private String name;
     private Double price;
-    private LocalDateTime cookedAt;
+    private Integer cookingTime;
```

Comment:
Bad idea, we already have discussed why

Reply:
Ops, I will change the type from Integer to Period.